### ISSUE TITLE
A11y support for tables

### ISSUE THREAD

**Issue by dlabaj** [patternfly/patternfly-next#562]
Investigate and come up with POC for a11y support for tables.

**Comment by mattnolting** [comment]
Findings:
- For interactive tables, assigning role="grid" is appropriate, as described by https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html. This is unnecessary for basic tables. 
- Tables, featuring expandable content do not conform to role="treegrid" usage as described by https://www.w3.org/TR/2018/WD-wai-aria-practices-1.2-20180719/examples/treegrid/treegrid-1.html as the data within expandable rows does not relate directly to specific table header cells.
- `aria-expanded="true/false"' should be assigned to the `<tr>` being expanded/collapsed
- `aria-sort="ascending/descending/none"' should be assigned to sortable `<th>`
  - `none` is used when the `<th>` is sortable, but not actively sorting
- `aria-hidden="true"` should be assigned to collapsed `<tr>` and `<th>`, `<td>`

...still adding to this
